### PR TITLE
enable ORTModelForSpeechSeq2Seq optimization

### PR DESCRIPTION
**Pull request by flozi00**
# What does this PR do?

Fixes # (issue)

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you make sure to update the documentation with your changes?
- [ ] Did you write any new necessary tests?

### CODE REVIEW

**Comment by flozi00** [comment]
@michaelbenayoun @echarlaix

File: optimum/onnxruntime/optimization.py [hunk 1]
```diff
@@ -74,7 +74,7 @@ def from_pretrained(
         onnx_model_path = []
         config = None
         if isinstance(model_or_path, ORTModel):
-            if isinstance(model_or_path, ORTModelForSeq2SeqLM):
+            if isinstance(model_or_path, ORTModelForSeq2SeqLM) or isinstance(model_or_path, ORTModelForSpeechSeq2Seq):
```
Comment (michaelbenayoun) [inline optimum/onnxruntime/optimization.py:77]:
```suggestion
            if isinstance(model_or_path, ORTModelForConditionalGeneration):
```

Comment (michaelbenayoun) [reply optimum/onnxruntime/optimization.py:77]: I think this would be the most general case here, @echarlaix wdyt?

**Comment by michaelbenayoun** [comment]
There is a bug because you did not import `ORTModelForConditionalGeneration` in `optimization.py`.
You need to add this and also please remove unused imports at the same time! (if there is any)

**Pull request closed without merging**